layers/+lang/flow-type/README.org

```diff
@@ -2,11 +2,18 @@
 
 #+TAGS: framework|layer|programming
 
+* Table of Contents                     :TOC_5_gh:noexport:
+- [[#table-of-contents][Table of Contents]]
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#install][Install]]
+- [[#key-bindings][Key bindings]]
+
 * Table of Contents
```

Comment:
@smile13241324  this has to be removed.  It didn't have  `:TOC_5_gh:noexport:` so the bot treated this TOC as a usual headline.